File: ios-oc/VoiceAgent/KeyCenter.m
//
//  KeyCenter.m
//  VoiceAgent
//
//  Created by qinhui on 2025/11/17.
//

#import "KeyCenter.h"

static NSString *const AG_APP_ID = @"";
static NSString *const AG_APP_CERTIFICATE = @"";
static NSString *const LLM_API_KEY = @"";
static NSString *const LLM_URL = @"https://api.deepseek.com/v1/chat/completions";
static NSString *const LLM_MODEL = @"deepseek-chat";
static NSString *const TTS_BYTEDANCE_APP_ID = @"";
static NSString *const TTS_BYTEDANCE_TOKEN = @"";

@implementation KeyCenter

+ (NSString *)AG_APP_ID {
    return AG_APP_ID;
}

+ (NSString *)AG_APP_CERTIFICATE {
    return AG_APP_CERTIFICATE;
}

+ (NSString *)LLM_API_KEY {
    return LLM_API_KEY;
}

+ (NSString *)LLM_URL {
    return LLM_URL;
}

+ (NSString *)LLM_MODEL {
    return LLM_MODEL;
}

+ (NSString *)TTS_BYTEDANCE_APP_ID {
    return TTS_BYTEDANCE_APP_ID;
}

+ (NSString *)TTS_BYTEDANCE_TOKEN {
    return TTS_BYTEDANCE_TOKEN;
}

@end

File: ios-swift/VoiceAgent/KeyCenter.swift
//
//  KeyCenter.swift
//  VoiceAgent
//
//  Created by qinhui on 2025/11/3.
//

import Foundation

class KeyCenter {
    // Agora Configuration
    static let AG_APP_ID: String = ""
    static let AG_APP_CERTIFICATE: String = ""

    // LLM - DeepSeek
    static let LLM_API_KEY: String = ""
    static let LLM_URL: String = "https://api.deepseek.com/v1/chat/completions"
    static let LLM_MODEL: String = "deepseek-chat"

    // STT - Microsoft Azure
    static let STT_MICROSOFT_KEY: String = ""
    static let STT_MICROSOFT_REGION: String = "chinaeast2"

    // TTS - MiniMax
    static let TTS_MINIMAX_KEY: String = ""
    static let TTS_MINIMAX_MODEL: String = "speech-01-turbo"
    static let TTS_MINIMAX_VOICE_ID: String = "male-qn-qingse"
    static let TTS_MINIMAX_GROUP_ID: String = ""
}
